tools_docker/test_docker/README.md

```diff
@@ -0,0 +1,44 @@
+# Docker Tests
+
+This is a too to help test docker, for simple case, but also for complicated test.
+This allow:
+- to run automatically tracim, collabora and elasticsearch with by defaut a working case where all features of tracim work (it just need some tweak
+for email and you need to setup resolving using special docker)
+- to test if docker start correctly
+- to test automatically service running in the docker
+
+## Prerequisities
+
+- pip install pytest testinfra python-dotenv
+- build tracim docker image "DOCKER_TRACIM_IMAGE" (see test_config.env)
+- build elasticsearch docker image "DOCKER_ELASTICSEARCH_IMAGE"
+- cp tracim.sample.env tracim.env
+- cp test_config.sample.env test_config.env
```

Comment:
```suggestion
~~~sh
pip install pytest testinfra python-dotenv
build tracim docker image "DOCKER_TRACIM_IMAGE" (see test_config.env)
build elasticsearch docker image "DOCKER_ELASTICSEARCH_IMAGE"
cp tracim.sample.env tracim.env
cp test_config.sample.env test_config.env
~~~
```
(not sure about the `~~~` syntax, but the 3 backticks don't work inside this suggestion)

Reply:
these are not shell command but actions to do. You can't just set this to shell script.